### PR TITLE
Allow trailing comma in JSDoc record types.

### PR DESCRIPTION
**Pull request by rintaro**
Fixes: #796

### CODE REVIEW

File: src/com/google/javascript/jscomp/parsing/JsDocInfoParser.java [hunk 1]
```diff
@@ -2383,6 +2383,12 @@ private Node parseFieldTypeList(JsDocToken token) {
 
       // Move to the token passed the comma.
```
Comment (dimvar) [inline src/com/google/javascript/jscomp/parsing/JsDocInfoParser.java]: Change to

// Move to the token past the comma

File: src/com/google/javascript/jscomp/parsing/JsDocInfoParser.java [hunk 2]
```diff
@@ -2383,6 +2383,12 @@ private Node parseFieldTypeList(JsDocToken token) {
...
       skipEOLs();
+
+      if (match(JsDocToken.RIGHT_CURLY)) {
+        // Allow trailing comma.
```
Comment (dimvar) [inline src/com/google/javascript/jscomp/parsing/JsDocInfoParser.java]: Change to

// Allow trailing comma (ie, right curly following the comma)

**Comment by dimvar** [comment]
Looks good, just a couple minor comments. After you do the fixes, please squash the commits and reply here and I'll merge the PR.

**Comment by rintaro** [comment]
Thanks, done.

**Pull request merged**